devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy

```diff
@@ -20,29 +20,57 @@ metadata {
 		capability "Switch"
 		capability "Health Check"
 
+		// Generic
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006"
+		fingerprint profileId: "C05E", deviceId: "0000", inClusters: "0006", deviceJoinName: "Generic On/Off Light"
```

Comment:
Did you want the profile for ZLL in here?  Shouldn't this one be HA?

Reply:
We don't have a ZLL Switch DTH and there was already a ZLL device in this DTH so I thought it might work. I can do a little more digging though. What I might do is remove all the new generic fingerprints from this PR so it's just the formatting changes and then do a second set of PRs for the generic fingerprints.